.github/workflows/release-action-docker.yml

```diff
@@ -0,0 +1,56 @@
+name: Release Github action for CLI
+
+on:
+  release:
+    types:
+      - prereleased
```

Comment:
I believe this only works when you do a manual pre-release.
![image](https://github.com/user-attachments/assets/e268c2ef-0891-4f6f-b5ab-f18c7ba71244)

Reply:
Yeah, so you can then proceed with the release with the release, once the workflow is successful, right?

Reply:
But the release is no longer automated if it has to be done manually. Let me correct the misunderstanding pre-release event will only work when a pre-release is done, which is done nowhere as far as I know.